gio/src/list_store.rs

```diff
@@ -52,6 +60,37 @@ impl<O: IsA<ListStore>> ListStoreExtManual for O {
             )
         }
     }
+
+    #[cfg(any(feature = "v2_64", feature = "dox"))]
+    #[cfg_attr(feature = "dox", doc(cfg(feature = "v2_64")))]
+    fn find_with_equal_func<P: IsA<glib::Object>>(
+        &self,
+        item: &P,
+    ) -> Option<u32> {
+        unsafe extern "C" fn equal_func_trampoline(
+            a: glib::ffi::gconstpointer,
+            b: glib::ffi::gconstpointer,
+        ) -> glib::ffi::gboolean {
+            let a: Borrowed<glib::Object> = from_glib_borrow(a as *mut glib::gobject_ffi::GObject);
+            let b: Borrowed<glib::Object> = from_glib_borrow(b as *mut glib::gobject_ffi::GObject);
+        
+            (*a == *b).into_glib()
```

Comment:
Just noticed that this doesn't actually make use of a compare_func. The `find_with_equal_func` is supposed to take a function used to find the item you're looking for. There's already a `find` function that works without having to pass a compare callback. 

The issue we have here and the reason why this wasn't automatically generated I guess is the original C function doesn't provide a way for passing the compare callback as user_data.

Reply:
See https://gitlab.gnome.org/GNOME/glib/-/issues/2447

Reply:
You could in theory work around this by storing the closure in TLS but...